pages/features/online-payments.vue

```diff
@@ -0,0 +1,279 @@
+          >
+            <h1 class="section__heading">Secure Online Payments</h1>
+            <p class="hero__subheading mt-3">
+              Formester’s online forms with payment integration make receiving payments more convenient than ever with secure PCI compliance, responsive interface, and no additional charges.
+            </p>
+            <a
+              href="https://app.formester.com/users/sign_up"
+              class="hero__invert__button mt-4"
+              >Get Started - it's Free</a
+            >
+          </div>
+          <div
+            class="col-lg-7 d-flex align-items-center justify-content-center mt-lg-0 mt-5"
+          >
+            <nuxt-img
+              src="/features/online-payment/online-payment-hero.svg"
+              alt="Hero-Image"
+              class="img-fluid hero__image"
+            />
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="container py-5">
+      <h2 class="section__heading">
+        Automate Payment Collection and Order Processing
+      </h2>
+      <div
+          class="d-flex flex-column flex-md-row justify-content-between mt-md-5"
+        >
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/online-payment/track-payments.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">Track Payments</h4>
+            <p class="subheading__text">
+              Keep a track of your payment data along with live updates when a payment is processed.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/online-payment/all-in-one-solution.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">All in One Solution</h4>
+            <p class="subheading__text">
+              Regardless of your industry, formester provides you with end-to-end solutions with a comprehensive suite of features.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-column align-items-center align-items-md-start text-center text-md-start px-4 col-md-4 my-3 mt-lg-5"
+          >
+            <nuxt-img
+              style="width: 69px"
+              src="/features/online-payment/integrations.svg"
+              alt="Formester"
+            />
+            <h4 class="sub__section-heading mt-4">Integrations</h4>
+            <p class="subheading__text">
+              Select the payment gateway of your choice from our 35+ payment integrations.
+            </p>
+          </div>
+        </div>
+    </div>
```

Comment:
same here there should be a component for this.

Reply:
Okay @dhaladitya108